docs/emigrate-nobles.rst

```diff
@@ -0,0 +1,36 @@
+emigrate-nobles
+===============
+
+.. dfhack-tool::
+    :summary: Allow inherited nobles to emigrate from fort to their site of governance.
+    :tags: fort units
+
+Tired of inherited nobility freeloading off your fortress making inane demands? Use this tool
+to have them (willingly) emigrate to their rightful lands.
+
+The unit must not be assigned to a squad.
+
+Warning! Nobles will not surrender any assigned items, be sure to unassign your artefacts before
```

Comment:
maybe mention "symbols of office" specifically here. Otherwise people might just think you're referring to the clothes they're wearing or the military equipment they have equipped.

Reply:
Will do, I wasn't sure what the right term for this was 😅. Thanks!